lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -1613,6 +1613,96 @@ export class BrowserCacheManager extends CacheManager {
         this.setInteractionInProgress(false);
     }
 
+    /**
+     * Create request retry key to cache retry status
+     * @param correlationId
+     */
+    generateRequestRetriedKey(correlationId: string): string {
+        return `${Constants.CACHE_PREFIX}.${TemporaryCacheKeys.REQUEST_RETRY}.${correlationId}`;
+    }
+
+    /**
+     * Gets the request retry value from the cache
+     * @param correlationId
+     */
+    getRequestRetried(correlationId: string): string | null {
+        const requestRetriedKey = this.generateRequestRetriedKey(correlationId);
+        return this.getTemporaryCache(requestRetriedKey);
+    }
+
+    /**
+     * Sets the request retry value to "retried" in the cache
+     * @param correlationId
+     */
+    setRequestRetried(correlationId: string): void {
+        this.logger.trace("BrowserCacheManager.setRequestRetried called");
+        const requestRetriedKey = this.generateRequestRetriedKey(correlationId);
+        this.setTemporaryCache(requestRetriedKey, "retried", false);
```

Comment:
Can we use a boolean or a 0/1 instead of a string here?

Reply:
Hmmm setTemporaryCache takes in a string and it's cached as a string. Is there a benefit to having it be a boolean or 0/1? The usage is checking whether the requestRetried cache item for the correlationId is null or not.

Reply:
Magic strings aren't great practice, if we're treating the value as a simple true/false we should pick a value that is consistent with that - a stringified "1" or "true" would be OK, I'd personally lean towards the numerical in case we ever want to retry more than once, but I'll leave it up to you.

Reply:
Makes sense, thanks. Updated to stringified "1".